.github/legal/cla-individual.md

```diff
@@ -0,0 +1,91 @@
+# Open Instance Power System Library - OpenIPSL \ Individual Contributor License Agreement
+
+Thank you for your interest in contributing to Open Instance Power System Library - OpenIPSL  ("We" or "Us").
+
+This contributor agreement ("Agreement") documents the rights granted by contributors to Us. To make this document effective, please sign it and send it to Us by electronic submission, following the instructions [here](README.md). This is a legally binding document, so please read it carefully before agreeing to it. The Agreement may cover more than one software project managed by Us.
```

Comment:
```suggestion
This contributor agreement ("Agreement") documents the rights granted by contributors to Us. To make this document effective, please sign it and send it to Us by electronic submission, following the instructions [README](../../README.md). This is a legally binding document, so please read it carefully before agreeing to it. The Agreement may cover more than one software project managed by Us.
```

Reply:
Better double check if the link is working. Relative links can be funny when looking at source location and deployment location.

Reply:
Same here.